chapters/en/chapter6/pre-trained_models.mdx

```diff
@@ -0,0 +1,330 @@
+audio, and the discriminator can easily differentiate it from real audio. As training progresses, the generator improves 
+its output, aiming to fool the discriminator. The discriminator, in turn, becomes more accurate in distinguishing real 
+and generated audio. This adversarial feedback loop helps both networks improve over time. Ultimately, HiFi-GAN learns to 
+generate high-fidelity audio that closely resembles the characteristics of the training data.
+
+</Tip>
+
+Loading the vocoder is as easy as any other 🤗 Transformers model.
+
+```python
+from transformers import SpeechT5HifiGan
+
+vocoder = SpeechT5HifiGan.from_pretrained("microsoft/speecht5_hifigan")
+```
+
+Now all you need to do is pass it as an argument when generating speech, and the outputs will be automatically converted to the speech waveform.
+
+```python
+speech = model.generate_speech(inputs["input_ids"], speaker_embeddings, vocoder=vocoder)
+```
+
+Let's listen to the result. The sample rate used by SpeechT5 is always 16 kHz.
+
+```python
+from IPython.display import Audio
+
+Audio(speech, rate=16000)
+```
+
+Neat! 
+
+Feel free to play with the SpeechT5 text-to-speech demo, explore other voices, experiment with inputs. Note that this 
+pre-trained checkpoint only supports English language:
+
+<iframe
+	src="https://matthijs-speecht5-tts-demo.hf.space"
+	frameborder="0"
+	width="850"
+	height="450">
+</iframe>
+
+### Massive Multilingual Speech (MMS)
+
+What if you are looking for a pre-trained model in a language other than English? Massive Multilingual Speech (MMS) is 
+another model that covers an array of speech tasks, however, it supports a large number of languages. For instance, it can 
+synthesize speech in over 1,100 languages.
+
+MMS for text-to-speech is based on [VITS Kim et al., 2021](https://arxiv.org/pdf/2106.06103.pdf), which is one of the 
+state-of-the-art TTS approaches.
+
+VITS is a speech generation network that converts text into raw speech waveforms. It works like a conditional variational 
+auto-encoder, estimating audio features from the input text. First, acoustic features, represented as spectrograms, are 
+generated. The waveform is then decoded using transposed convolutional layers adapted from HiFi-GAN. 
+During inference, the text encodings are upsampled and transformed into waveforms using the flow module and HiFi-GAN decoder.
+This means that you do not need to add a vocoder for inference, it's already "built-in".  
+
+<Tip warning={true}>
+	
+MMS model has been added to 🤗 Transformers very recently, so the API may still slightly change. At the time of this Unit's 
+release, the MMS for TTS is scheduled to be merged in Transformers `main` in a couple of days. 
+Once it's merged, you will have to install the library from source, if you'd like to try it out:
+
+```bash
+pip install git+https://github.com/huggingface/transformers.git
```

Comment:
Shall we install from the branch in this tip so that people can run the following code already? 
```suggestion
pip install git+https://github.com/hollance/transformers.git@vits
```